docs/user_manual/calculations.md

```diff
@@ -85,6 +85,20 @@ The [iterative linear](#iterative-linear-state-estimation) and [Newton-Raphson](
 In observable systems this helps better outputting correct results. On the other hand with unobservable systems, exceptions raised from calculations due to faulty results will be prevented.
 ```
 
+##### Necessary observability condition
```

Comment:
i think this is the same as the above documentation (line 51-81 of this file on this branch)

Reply:
That explanation is from theory POV. The assumption and combining of sensor is not mentioned.